src/generator.cs

```diff
@@ -3419,7 +3631,12 @@ void GenerateTypeLowering (MethodInfo mi, bool null_allowed_override, EnumMode e
 				disposes.AppendFormat (GenerateDisposeString (probe_null, !mai.ZeroCopyStringMarshal), pi.Name);
 			} else if (mai.Type.IsArray){
 				Type etype = mai.Type.GetElementType ();
-				if (etype == typeof (string)){
+				if (HasBindAsAttribute (pi)) {//Alex
```

Comment:
hahaha yeah I use that to quickly navigate the src file, I wish XS supported multiple windows of the same file